service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/sendtogatekeeper/NotifyGatekeeperTemplate.java

```diff
@@ -1,28 +1,28 @@
 package uk.gov.hmcts.reform.fpl.model.notify.sendtogatekeeper;
 
-import lombok.Getter;
-import lombok.Setter;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.experimental.SuperBuilder;
 import uk.gov.hmcts.reform.fpl.model.notify.SharedNotifyTemplate;
 
-@Getter
-@Setter
+@EqualsAndHashCode(callSuper = true)
+@Data
+@SuperBuilder
 public final class NotifyGatekeeperTemplate extends SharedNotifyTemplate {
-    public NotifyGatekeeperTemplate duplicate() {
-        NotifyGatekeeperTemplate clone = new NotifyGatekeeperTemplate();
-
-        clone.setLocalAuthority(this.getLocalAuthority());
-        clone.setOrdersAndDirections(this.getOrdersAndDirections());
-        clone.setDataPresent(this.getDataPresent());
-        clone.setFullStop(this.getFullStop());
-        clone.setTimeFramePresent(this.getTimeFramePresent());
-        clone.setTimeFrameValue(this.getTimeFrameValue());
-        clone.setUrgentHearing(this.getUrgentHearing());
-        clone.setNonUrgentHearing(this.getNonUrgentHearing());
-        clone.setFirstRespondentName(this.getFirstRespondentName());
-        clone.setReference(this.getReference());
-        clone.setCaseUrl(this.getCaseUrl());
-
-        return clone;
+    public SharedNotifyTemplate duplicate() {
+        return SharedNotifyTemplate.builder()
```

Comment:
Can just be `this.toBuilder().build()`

Reply:
I tried it but it's causing a code smell suggesting to access the static builder. sonarcloud reported the same earlier